src/lib/nvme_pt_ioctl.c

```diff
@@ -328,6 +328,8 @@ void opal_deinit_pt(struct sed_device *dev)
 	}
```

Comment:
do we need to set discv pointer to NULL after freeing discv?

Reply:
I'm not sure if it makes any difference

Reply:
Yeah this can avoid a dangling pointer.